tests/utils/iotrace.py

```diff
@@ -382,9 +423,7 @@ def remove_traces(prefix: str, force: bool = False, shortcut: bool = False):
             command += ' -f ' if shortcut else ' --force '
 
         output = TestRun.executor.run(command)
-        error_output = parse_json(output.stderr)[0]['trace']
-        if error_output == "No traces removed.":
-            raise CmdException("Invalid traces removal", output)
+        return parse_json(output.stderr)
```

Comment:
Please return stdout. In case of error when getting message "No traces removed" return empty json "{}"

Reply:
Returning empty string, since functions return parsed json